dist/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To Do List</title>
+  <script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+  <body>
+    <section class="container">
+      <ul>
+        <li class="header flex list">
+          <h2>Today's To Do</h2>
+          <button class="borderless pointer" id="refreshBtn">
+            <i class="fas fa-sync-alt normal-icons pointer"></i>
+          </button>
+        </li>
+        <li class="task-input-container flex list">
+          <form action="" data-new-task-form>
+            <input
+              type="text"
+              class="task-input"
+              data-new-task-input
+              placeholder="Add to your list..."
+            />
+            <button class="borderless" id="addTaskBtn">
```

Comment:
It would be better if you added the type of the button, for instance type="submit" as this is a button that submits a form.